src/components/pagination/pagination.styles.tsx

```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
+
+export const StylesWrapper = styled.div<{}>`
```

Comment:
Remember you can declare css variables with values and consume them in the CSS 🦆

Reply:
🥔 Done! I've updated the styles file and use css variables